jps-plugin/src/org/jetbrains/jps/incremental/scala/data/CompilationData.scala

```diff
@@ -83,10 +85,11 @@ object CompilationData {
         if (hydraSettings.isHydraEnabled && scalaVersion.nonEmpty && hydraGlobalSettings.containsArtifactsFor(scalaVersion.get, hydraSettings.getHydraVersion))
           Seq("-sourcepath", outputGroups.map(_._1).mkString(File.pathSeparator), "-cpus", hydraSettings.getNumberOfCores,
             "-YsourcePartitioner:" + hydraSettings.getSourcePartitioner, "-YhydraStore", hydraSettings.getHydraStorePath,
-            "-YpartitionFile", hydraSettings.getHydraStorePath + module.getName, "-YrootDirectory", hydraSettings.getProjectRoot,
+            "-YpartitionFile", hydraSettings.getHydraStorePath + File.separator + module.getName, "-YrootDirectory", hydraSettings.getProjectRoot,
```

Comment:
The path for the partition file was correct before, and I'm afraid not it may contain a double separator. My advise would be to use a NIO `Path`  when concatenating paths if the minimum required version for IJ is Java7 (if it's Java6 then use a `File`).